frontend/jsontablemodel.cpp

```diff
@@ -119,6 +119,8 @@ QVariant JsonTableModel::data(const QModelIndex &index, int role) const
                                        QString::number(minutes).rightJustified(2, '0') + ":" +
                                        QString::number(seconds).rightJustified(2, '0'));
                     }
+                    if(QDateTime::fromString(v.toString(), "yyyy-MM-dd'T'HH:mm:ss").isValid())
+                        return QDateTime::fromString(v.toString(), "yyyy-MM-dd'T'HH:mm:ss").toString("dd/MM/yyyy HH:mm:ss");
```

Comment:
I was going to complain here that you didn't add specific "Today" and "Yesterday" cases, but obviously that's hard to test until we start cycle!  Can make a note to come back to this issue.

Reply:
I can implement this for now and then when I get round to unit testing we should be able to assess this